packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -5625,3 +5629,69 @@ export const getProjectStorageUsage = (
     { signal },
   )
 }
+
+// https://rest-docs.synapse.org/rest/POST/grid/session/async/start.html
+// https://rest-docs.synapse.org/rest/GET/grid/session/async/get/asyncToken.html
+export const GridSessionAsyncStart = async (
```

Comment:
Some naming suggestions since these are functions

```suggestion
export const startGridSession = async (
```